repositories/src/main/scala/cromwell/pipeline/datastorage/dto/WomToolNode.scala

```diff
@@ -0,0 +1,43 @@
+package cromwell.pipeline.datastorage.dto
+
+import play.api.libs.json._
+
+final case class WomToolNode(name: String, value: Option[String])
+
+object WomToolNode {
+  implicit object WomToolNodeFormat extends Format[WomToolNode] {
+    override def writes(o: WomToolNode): JsValue = JsObject(Seq(o.name -> JsString(optToString(o.value))))
+
+    override def reads(json: JsValue): JsResult[WomToolNode] = {
+      val (name, value) = json.as[(String, String)]
+      JsSuccess(WomToolNode(name, stringToOpt(value)))
+    }
+  }
+
+  private def optToString(opt: Option[String]): String =
+    opt.getOrElse("null")
```

Comment:
This looks bad. You wrote your own `OptionFormat` class (and I still don't understand why did you do this), why are you using another logic to serialize/deserialize `Option` here?

Reply:
Yes, thanks for notice that, deleted it immediately! :)